lib/topologies/mongos.js

```diff
@@ -467,6 +468,12 @@ function connectProxies(self, servers) {
       server.once('parseError', handleInitialConnectEvent(self, 'parseError'));
       server.once('error', handleInitialConnectEvent(self, 'error'));
       server.once('connect', handleInitialConnectEvent(self, 'connect'));
+
+      // Command Monitoring events
+      server.on('commandStarted', event => self.emit('commandStarted', event));
```

Comment:
Can we make a shared function like this:

```js
function addCommandMonitoringListeners(listener, emitter) {
  ['commandStarted', 'commandSucceeded', 'commandFailed'].forEach(eventName => {
    listener.on(eventName, event => emitter.emit(eventName, event));
  });
}
```

Reply:
sure